### ISSUE TITLE
Add Snackbar component

### ISSUE THREAD

**Issue by mnicole** [gumgum/gumdrops#6]
http://ds.gumgum.com/stable/#gds-snackbar

**Comment by luizmendez** [comment]
Can I take this issue? Just created a Snackbar component in a recent project that I may be able to reuse.

**Comment by luizmendez** [comment]
@jscottsmith Thanks, the docs are really great but if something comes up I'll let you know.

**Comment by luizmendez** [comment]
Also I noted that the [snackbar in your documents](https://ds.gumgum.com/stable/md/docs/snackbar-component.html) is a wrapper for snackbar notifications, I think this should be worked as two different components: `<Snackbar>` and `<SnackbarNotification>`.

The user could pass as children the `<SnackbarNotification>` needed to the <Snackbar> wrapper component:

```
<Snackbar>
    <SnackbarNotification>Text Message here</SnackbarNotification>
    <SnackbarNotification context='warning' text='A warning message'>
<Snackbar/>
```

**Comment by jscottsmith** [comment]
@luizmendez 

> I think this should be worked as two different components: <Snackbar> and <SnackbarNotification>.

Agreed!

> implementing these extra options is ok or would you rather just have the basic functionality of the component as described in the docs.

- `timeToClose {number}`
- `hideClose {bool}`
- `text {string}`
- `callback {func}`

Yes, all those options will be needed, thanks!

**Comment by luizmendez** [comment]
@jscottsmith Great, thanks!

**Comment by edrpls** [comment]
Closed on #67
